### PR TITLE
checks: Update encoder overload guide links

### PR DESCRIPTION
**Pull request by prgmitchell**
### Description

Update any references to the GPU overload guide to the KB article here: https://obsproject.com/kb/encoding-performance-troubleshooting

### Motivation and Context

The old link just references this one, this was mentioned in the community discord.

### How Has This Been Tested?

Just confirmed the link is correct.

### Types of changes

- Bug fix (non-breaking change which fixes an issue) 

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: checks/encoding.py [hunk 1]
```diff
@@ -131,16 +131,16 @@ def checkEncoding(lines):
             severity = LEVEL_INFO
         if (hasx264 > 0 and (hasAMD + hasQSV + hasNVENC + hasAPPLE) > 0):
             return [severity, "{}% Encoder Overload".format(val),
-                    """Encoder overload may be related to your CPU or GPU being overloaded, depending on the encoder in question. If you are using a software encoder (x264) please see the <a href="https://obsproject.com/wiki/General-Performance-and-Encoding-Issues">CPU Overload Guide</a>. If you are using a hardware encoder (AMF, QSV/Quicksync, NVENC) please see the <a href="https://obsproject.com/wiki/GPU-overload-issues">GPU Overload Guide</a>."""]
+                    """Encoder overload may be related to your CPU or GPU being overloaded, depending on the encoder in question. If you are using a software encoder (x264) please see the <a href="https://obsproject.com/wiki/General-Performance-and-Encoding-Issues">CPU Overload Guide</a>. If you are using a hardware encoder (AMF, QSV/Quicksync, NVENC) please see the <a href="https://obsproject.com/kb/encoding-performance-troubleshooting">GPU Overload Guide</a>."""]
```
Comment (RytoEX) [inline checks/encoding.py:134]: We should probably also update `https://obsproject.com/wiki/General-Performance-and-Encoding-Issues`. If you want to do that in a separate PR or here, either is fine with me.

**Review by RytoEX — COMMENTED** [review]
We should probably also update `https://obsproject.com/wiki/General-Performance-and-Encoding-Issues` to `https://obsproject.com/kb/encoding-performance-troubleshooting`. That can either be added here, or done as a separate PR.

File: checks/encoding.py (hunk 1 above)
Comment (prgmitchell) [reply checks/encoding.py:134]: I will create a separate PR for this sometime this week.

**Review by RytoEX — APPROVED** [review]
